fvm/src/call_manager/default.rs

```diff
@@ -351,6 +351,15 @@ where
         // it returns a referenced copy.
         let engine = self.engine().clone();
 
+        // prepare actor code
+        // TODO remove once the EVM smart contract actor is integrated in the builtin-actors bundle
+        //  https://github.com/filecoin-project/ref-fvm/issues/693
+        self.engine()
+            .load_code_cid(&state.code, self.blockstore())
+            .map_err(
+                |_| syscall_error!(NotFound; "actor code cid does not exist {}", &state.code),
+            )?;
+
```

Comment:
Tracked in #708.

Reply:
Isn't this part of a separate patch?

- Introduce a secp recover syscall.
- Code loading?